src/server/set_family.cc

```diff
@@ -1020,6 +1030,36 @@ void SIsMember(CmdArgList args, ConnectionContext* cntx) {
   }
 }
 
+void SMIsMember(CmdArgList args, ConnectionContext* cntx) {
+  string_view key = ArgS(args, 1);
+
+  vector<string_view> vals(args.size() - 2);
+  for (size_t i = 2; i < args.size(); ++i) {
+    vals[i - 2] = ArgS(args, i);
+  }
+
+  auto cb = [&](Transaction* t, EngineShard* shard) {
+    OpResult<PrimeIterator> find_res = shard->db_slice().Find(t->db_context(), key, OBJ_SET);
+    if (find_res) {
+      SetType st{find_res.value()->second.RObjPtr(), find_res.value()->second.Encoding()};
+      return OpInSet(t->db_context(), st, vals);
+    }
+    if (find_res.status() == OpStatus::KEY_NOTFOUND) {
```

Comment:
i prefer that instead of handling special cases like `KEY_NOTFOUND` here, you will handle it at the caller side (after ScheduleSingleHopT).  See other methods that do that.

Reply:
Refactored in https://github.com/dragonflydb/dragonfly/pull/377/commits/5627a795a7c99bca016a812b3ab9d3025b62041c